sonar-csharp-plugin/pom.xml

```
@@ -100,7 +100,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>19.0</version>
+      <version>[24.1.1,)</version>
```

Why not the [latest](https://github.com/google/guava/releases)? As also used in https://github.com/SonarSource/sonar-java/blob/26910434c68fd9ece1c0e6f898112c30833f2dad/pom.xml#L153-L157
```suggestion
      <version>30.1-jre</version>
```